components/ServiceItem.tsx

```diff
@@ -22,56 +22,58 @@ export const ServiceItem = ({
   ...props
 }: ServiceProps): JSX.Element => {
 
+
+
+
+
   return (
     <Box {...props}>
-      <LinkBox mb="16px">
-        <HStack spacing="8px">
-          <LinkIcon />
-          <LinkOverlay href={service.url}>
-            <Heading fontSize="subheading2">{service.name}</Heading>
-          </LinkOverlay>
-        </HStack>
-      </LinkBox>
-      <Stack spacing="8px">
-        <Text>{service.description}</Text>
-        {service.email && (
-          <LinkBox>
-            <HStack spacing="8px">
-              <EmailIcon />
-              <LinkOverlay href={`mailto:${service.email}`}>
-                {service.email}
-              </LinkOverlay>
-            </HStack>
-          </LinkBox>
-        )}
-        {service.phoneNumbers && (
-          <LinkBox>
-            <HStack spacing="8px">
-              <PhoneIcon />
-              <LinkOverlay href={`tel:${service.phoneNumbers[0]}`}>
-                {service.phoneNumbers[0]}
-              </LinkOverlay>
-            </HStack>
-          </LinkBox>
-        )}
-        {service.taxonomyString && (
-          <HStack>
-            <Heading fontSize="subheading3">
-              Resource categories:
-            </Heading>
-            {service.taxonomyString.map((taxonomy, i) => (
-              <Text
-                key={i}
-                bgColor="lightPink"
-                borderRadius="8px"
-                p="8px"
-              >
-                {taxonomy}
-              </Text>
-            ))}
+      <Box boxShadow='md' rounded='lg' p='8' _hover={{ border: '2px', borderColor: 'teal' }} >
```

Comment:
You can just pass the styling props to the top level `Box` component after the spread props `{...props}` instead having two `Box` components